### PR TITLE
[DOCS] Update Installation Guide - GPU steps

### PR DESCRIPTION
**Pull request by aalborov**

### CODE REVIEW

File: docs/install_guides/installing-openvino-linux.md [hunk 1]
```diff
@@ -320,16 +320,21 @@ cd /opt/intel/openvino/install_dependencies/
...
-
+sudo -E ./install_NEO_OCL_driver.sh
+```
+The drivers are not included in the package and the script downloads them. Make sure you have the 
+internet connection for this step.
+> **NOTES**: 
+> * If the driver version on the system is higher or equal to the current version, the script does 
+> not install a new driver. 
+> * If the version of the driver is lower than the current version, the script uninstalls the lower 
+> and installs the current version with your permission.
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-linux.md:333]: the screen shot goes here

File: docs/install_guides/installing-openvino-linux.md [hunk 2]
```diff
@@ -320,16 +320,21 @@ cd /opt/intel/openvino/install_dependencies/
...
 sudo -E su
 ```
-3. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics:
+3. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics. Run the installation script with super user rights:
 ```sh
-./install_NEO_OCL_driver.sh
-```
-You may see the following command line output:
-
+sudo -E ./install_NEO_OCL_driver.sh
```
Comment (aalborov) [inline docs/install_guides/installing-openvino-linux.md:325]: do we need `sudo -E` here if we entered superuser mode in the previous step?

**Review by artanokhov — APPROVED** [review]
Looks good to me
@evgenytalanin-intel do we need to describe something special about different driver versions?

**Comment by evgenytalanin-intel** [comment]
We need to specify that another driver versions are required for newer hardware, namely 20.35 but not 19.41. Not sure that we need to include this into driver installation script (as technically 19.41 is fine with Ice Lakes) - but anyway we need to explicitly state this.

File: docs/install_guides/installing-openvino-linux.md (hunk 2 above)
Comment (artanokhov) [reply docs/install_guides/installing-openvino-linux.md:325]: Actually not

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: docs/install_guides/installing-openvino-linux.md [hunk 3]
```diff
@@ -320,17 +320,23 @@ cd /opt/intel/openvino/install_dependencies/
...
+internet connection for this step.
+
+The script compares the driver version on the system to the current version. 
+If the driver version on the system is higher or equal to the current version, the script does 
+not install a new driver. 
+If the version of the driver is lower than the current version, the script uninstalls the lower 
+and installs the current version with your permission:
+![](../img/NEO_check_agreement.png)
+If the script fails to uninstall the driver, uninstall it manually.    
+You may see the following command line output:
```
[the remark this answers is not in the document]
Comment (aalborov) [reply docs/install_guides/installing-openvino-linux.md:337]: @artanokhov take a look please

[the remark this answers is not in the document]
Comment (artanokhov) [reply docs/install_guides/installing-openvino-linux.md:337]: >Is it during running the script 
This case

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**